logverification/verifylist_test.go

```diff
@@ -58,3 +60,38 @@ func TestVerifyListIntegration(t *testing.T) {
 	//  in the given list of events, i.e. omitted events. We can call the datatrails
 	//  events api to list events filtered by mmr index.
 }
+
+func TestVerifyListOmmitted(t *testing.T) {
+	logger.New("TestVerifyList")
+	defer logger.OnExit()
+
+	testContext, testGenerator, _ := integrationsupport.NewAzuriteTestContext(t, "TestVerifyList")
+	tenantID := mmrtesting.DefaultGeneratorTenantIdentity
+	generatedEvents := integrationsupport.GenerateTenantLog(
+		&testContext, testGenerator, 8, tenantID, true, integrationsupport.TestMassifHeight,
+	)
+	trimmedGeneratedEvents := generatedEvents[1:]
+	eventJsonList := serializeTestEvents(t, trimmedGeneratedEvents)
+	omittedIndices, err := VerifyList(testContext.Storer, eventJsonList)
+
+	require.Nil(t, err)
+	require.Len(t, omittedIndices, 1)
+}
+
+func TestVerifyListModifiedShouldError(t *testing.T) {
```

Comment:
nice! 

can we also have some excluded event cases?

i.e. where we add events into the eventjson, either at the same mmrIndex or at an intermediate mmrIndex